weave/flow/eval.py

```diff
@@ -248,6 +248,9 @@ async def predict_and_score(
                     """
                 )
                 raise OpCallError(message)
+            # this is a catchall in case the user's scorer function throws
+            except Exception as e:
+                result = {"error": str(e), "example": example}
```

Comment:
Don't reproduce the whole example here. That's going to really mess up with the auto-columns we show in the UI.

Can you make a realistic example with an LLM judge that tries to json.loads() an LLM call response, and has like 50 examples in the dataset? Those fail sporadically. Then look in the UI to understand the impact this change has on the user experience.

Reply:
I think this has some nice properties actually:
1. In the outputs, errors link to the exact rows that broke making debugging easier
2. Errors show as N/A in the UI which seems clean to me

![image](https://github.com/wandb/weave/assets/15385696/c513d062-cc30-4576-a6a9-925c920c15b8)
![image](https://github.com/wandb/weave/assets/15385696/43ae2676-8359-43f2-bcfd-074a8585fd1c)